docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc

```diff
@@ -0,0 +1,50 @@
+= Component deployment lifecycle
+CP Development Team <cp-devel@redhat.com>
+:toc: left
+:icons: font
+:numbered:
+:source-highlighter: highlightjs
+
+== Scope
+Learn how to track the lifecycle of a component's deployment in the {ProductName} web console.
+
+== Web console visualization
+In the *Overview* tab, {ProductName} shows the stages of a component deployment lifecycle. This visualization also shows the status of these stages, whether they are complete, running, or pending. 
+
+image::lifecycle_screenshot.png[alt=Component deployment lifecycle visualization]
+
+== Task run status list
+The following table explains the meaning of various icons that the web console shows during a component deployment lifecycle:
+
+[cols="1,1"]
+|===
+|Status |Symbol
+
+|Success
+|image:success_icon.svg[alt=Success]
+
+|Running
+|image:running_icon.svg[alt=Running]
+
+|Pending
+|image:pending_icon.svg[alt=Pending]
+
+|Failed
+|image:fail_icon.svg[alt=Failed]
+
+|Pipeline
+|image:pipeline_icon.svg[alt=Pipeline]
+
+|Environment (Static or Managed)
+|image:environment_icon.png[alt=Environment]
+
+|GitHub
+|image:github_icon.svg[alt=GitHub]
+
+|===
+
+== Build pipeline customization
+The lifecycle visualization is currently read-only. However, to customize the build pipelines that {ProductName} executes during this lifecycle, start by using the *Actions* drop-down menu to select *Customize build pipelines*. For detailed instructions on what to do next, visit the xref:getting-started/build_service.adoc[Build pipeline customization] page.
+
+== Logs
```

Comment:
Screenshot here as well?

Reply:
Created an issue to remind us to talk about this. I'd def like to add screenshots once we have a policy worked out for them!